Config.mak

```diff
@@ -3,11 +3,15 @@ export ASSERT_ON_STOMPING_PREVENTION=1
 
 override DFLAGS += -w -version=GLIBC
 
-ifeq ($(DVER),1)
-override DFLAGS := $(filter-out -di,$(DFLAGS)) -v2 -v2=-static-arr-params -v2=-volatile
-else
+ifeq ($(DVER),2)
 # Open source Makd uses dmd by default
 DC ?= dmd
```

Comment:
I don't get this, if we are in D2, then assign the same as makd uses by default?

Reply:
Ping @mihails-strasuns-sociomantic (I know it's not your fault ;-) ). This is the left over when we had `DC ?= dmd-transitional` here IIRC.

Reply:
Ah, right, it was just moved in this commit.